kanidmd/src/lib/core/ldaps.rs

```diff
@@ -21,31 +21,33 @@ use uuid::Uuid;
 
 struct LdapSession {
     uat: Option<LdapBoundToken>,
+    eventid: Uuid,
 }
 
 impl LdapSession {
-    fn new() -> Self {
+    fn new(eventid: &uuid::Uuid) -> Self {
```

Comment:
Clippy will be upset here because you are passing by reference a type that is copy ;)

Reply:
Clippy was fine with it!

Reply:
*gasp*